pandas/core/series.py

```diff
@@ -5630,7 +5630,7 @@ def dropna(
 
     def to_timestamp(
         self,
-        freq=None,
+        freq: str | None = None,
```

Comment:
I believe you can use `pandas._typing.Frequency` as this ends up in `PeriodArray.to_timestamp` which supports that.

Reply:
sounds good, will update